src/docfx/docset/Document.cs

```diff
@@ -141,21 +144,28 @@ internal class Document : IEquatable<Document>, IComparable<Document>
 
             _id = new Lazy<(string docId, string versionId)>(() => LoadDocumentId());
             _repository = new Lazy<Repository>(() => Docset.GetRepository(FilePath.Path));
+            _content = content;
 
+            Debug.Assert(!isFromHistory || _content != null);
```

Comment:
seems like the only case `isFromHistory` as `true` is get content from CRR git history? can we remove `IsFromHistory`, only keep `_content`?

Reply:
It's used in other place as well

Reply:
But it's a good suggestion, we can remove it.